### DISCUSSION TITLE
data analysis: What do you want to search and analyze in your data post-season?

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1788] [users: ideas for new features]
### Here we want to discuss what actual questions you want to address using your kiosk database. 

🎁 All answers and examples here will be immensely helpful during the design phase of the query module for Kiosk. 

Please be as concrete as possible. For instance:

- a list of all phase II contexts
- find all data where 'room II' or 'room 2' is mentioned
- all loci with seals
- in which locus did we find the leather thingy?

**Comment by urapadmin** [comment]
@lbestock wanted to solve the question:  "what contexts, even what URAP#s, had unfired or low fired pottery".

**Comment by lbestock** [comment]
Vague thought right now but coming out of looking at Athenian Agora digital research tools online and thinking about the visualization element: what role maps in here? We don't have spatial data in kiosk. Any analysis includes a spatial element.

**Reply by urapadmin** [reply]
> Vague thought right now but coming out of looking at Athenian Agora digital research tools online and thinking about the visualization element: what role maps in here? We don't have spatial data in kiosk. Any analysis includes a spatial element.

It is not important what we HAVE in Kiosk. The question here is what kind of questions do you want to ask your recording system.

**Comment by lbestock** [comment]
(Title of the ticket could be amended by adding "and what do you want the results to LOOK like".)

**Reply by urapadmin** [reply]
Let's not make that the title element. First there should be a question. What kind of visualization is reasonable can vary and I might have something to say about it.

**Comment by luizaogs** [comment]
My brain is a bit too dead right now to be truly helpful but these are a few things that come to mind right off the bat: 

- Which loci (or unit) include x material? 
- Which loci (or unit) include x type of small find? 
- What is the frequency of x material across different survey units?
- What is the frequency of x material across different sites? [thinking of Katie's project with different small rural sites, for instance]
- In which units did we excavate ovens [or another type of context]?
- What does the Harris Matrix for unit x look like? [don't kill me, that is something I would want to ask my recording system]

**Comment by urapadmin** [comment]
With the Harris matrix we want to:
- show all loci in the harris matrix that have certain types of materials in them and see what phase they belong to
